Revert "Fix iterator for ncclCommWatchdog. (#32571)"

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32649 Revert "Fix iterator for ncclCommWatchdog. (#32571)"**

This reverts commit 59dbece3716775c3e6f3a428f73fbf1bde8fac4f.

Revert "Enhance NCCL watchdog to acitvely abort communicators for timed out ops. (#32338)"

This reverts commit f86d6c6afd0e981642d20b4269837334ec46c140.

Differential Revision: [D19584224](https://our.internmc.facebook.com/intern/diff/D19584224)

ezyang committed:
Revert "Enhance NCCL watchdog to acitvely abort communicators for timed out ops. (#32338)"

This reverts commit f86d6c6afd0e981642d20b4269837334ec46c140.

[ghstack-poisoned]

pietern reviewed (approved):
Preemptive accept to unblock. What got broken?

mrshenli commented:
cc @pritamdamania87

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit ad220c97:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=5b321a098594693eca8408169f475cb334856655&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base 5b321a09 since Jan 27
  <details>
  <summary>
  Please rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=ad220c97ad52fcd325bed0ce73e5d127db23b98d).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_macos_10_13_py3_test](https://dr.pytorch.org/build-details.html?build_id=91283269) from Jan 27



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

ezyang commented:
Looks like I botched the merge

ezyang committed:
Update on "Revert "Fix iterator for ncclCommWatchdog. (#32571)""


This reverts commit 59dbece3716775c3e6f3a428f73fbf1bde8fac4f.

Revert "Enhance NCCL watchdog to acitvely abort communicators for timed out ops. (#32338)"

This reverts commit f86d6c6afd0e981642d20b4269837334ec46c140.

[ghstack-poisoned]

ezyang committed:
Update on "Revert "Fix iterator for ncclCommWatchdog. (#32571)""


This reverts commit 59dbece3716775c3e6f3a428f73fbf1bde8fac4f.

Revert "Enhance NCCL watchdog to acitvely abort communicators for timed out ops. (#32338)"

This reverts commit f86d6c6afd0e981642d20b4269837334ec46c140.

[ghstack-poisoned]
